buddyboss/buddyboss-platform
# fix bp_member_get_report_link function get_users slow query

### All Submissions:






### Changes proposed in this Pull Request:

This check was querying 'get_users' for more half seconds for sql query. so I have changed it to just check for user roles instead.

### How to test the changes in this Pull Request:

1. Go to any user profile
2. Check the query monitor and check slow queries
3. fix the issue.

### Other information:
